code/cfile/cfilearchive.h

```diff
@@ -23,6 +22,17 @@
 #define CFILE_BLOCK_UNUSED		0
 #define CFILE_BLOCK_USED		1
 
+struct COMPRESSION_INFO {
+	int header = 0;
+	size_t compressed_size = 0;
+	int block_size = 0;
+	int numOffsets = 0;
+	int* offsets = nullptr;
+	char* decoderBuffer = nullptr;
+	int lastDecBlockPos = 0;
+	int lastDecBytes = 0;
```

Comment:
You should probably decide on a naming scheme and stick with it. Please don't mix `underscore_names` and `camelCaseNames`. To stay consistent with existing code, I'd prefer underscores.